tests/bluetooth/audio/ascs/include/test_common.h

```diff
@@ -17,6 +17,22 @@ struct test_ase_chrc_value_hdr {
 	uint8_t params[0];
 } __packed;
 
+struct test_ase_cp_chrc_value_hdr {
+	uint8_t opcode;
+	uint8_t number_of_ases;
+	uint8_t params[0];
+} __packed;
+
+struct test_ase_cp_chrc_value_param {
+	uint8_t ase_id;
+	uint8_t response_code;
+	uint8_t reason;
+} __packed;
```

Comment:
Would something like
```suggestion

struct test_ase_cp_chrc_value_param {
	uint8_t ase_id;
	uint8_t response_code;
	uint8_t reason;
} __packed;

struct test_ase_cp_chrc_value_hdr {
	uint8_t opcode;
	uint8_t number_of_ases;
	struct test_ase_cp_chrc_value_param params[0];
} __packed;
```

make sense to do? Better indication of what the `params` actually refer to?